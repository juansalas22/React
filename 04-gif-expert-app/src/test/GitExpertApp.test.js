import { shallow } from 'enzyme';
import React from 'react';
import { GitExpertApp } from '../GitExpertApp';


describe('Pruebas en <GifExpertApp />', () => {
    
    test('Debe mostar <GitExpertApp /> correctamente ', () => {

        const wrapper = shallow(<GitExpertApp />);
        expect(wrapper).toMatchSnapshot();

    })
    
})
