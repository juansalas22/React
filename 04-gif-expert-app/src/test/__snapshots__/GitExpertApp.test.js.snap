// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifExpertApp /> Debe mostar <GitExpertApp /> correctamente  1`] = `
<Fragment>
  <h2>
    GitExpertApp
  </h2>
  <AddCategory
    setcategories={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category="Pokemon"
      key="Pokemon"
    />
  </ol>
</Fragment>
`;
